server/queues/request.js

```diff
@@ -0,0 +1,30 @@
+const Queue = require("bull");
+
+// initiate request queue
+// connect to redis default server for dev
+const requestQueue = new Queue("requestReview");
+
+// search reviewer, wait for 24 hours for response
+// rejected => search for another reviewer
+
+requestQueue.process(async (job) => {
+  const { languageLevel, status } = job.data;
```

Comment:
We will need the review/request model id, so we can fetch the review model and see the up to date status, and find a reviewer for that request